static/js/modules/helpers.js

```diff
@@ -118,6 +146,28 @@ function buildUrl(path, query) {
     .toString();
 }
 
+function getTimePeriod(electionYear, cycle, electionFull, office) {
+  var durations = {
+    P: 3,
+    S: 5,
+    H: 1
```

Comment:
Good question. Basically it's "[year after the last election] - [year of the next election]". Same logic as here:
![image](https://cloud.githubusercontent.com/assets/1696495/15303406/71a82fec-1b6c-11e6-91ff-e5fc01135d54.png)